### ISSUE TITLE
Make it possible to remove hitArea in TypeScript

### ISSUE THREAD

**Issue by mathiasbockwoldt** [pixijs/pixijs#8212]
### Expected Behavior
I would like to remove a previously set `hitArea` from a `Sprite` (or any `DisplayObject`) in TypeScript. 

### Current Behavior
In a new `Sprite`, the `hitArea` property seems to be `null`. After setting the `hitArea`, for example, to some `Rectangle`, I don't see a possibility to remove it again, since `hitArea` is defined as type `PIXI.IHitArea` (see [docs](https://pixijs.download/dev/docs/PIXI.DisplayObject.html#hitArea)). So if I try to set it to `null` (or `undefined`), I get the Compilation error `Type 'null' is not assignable to type 'IHitArea'.`.
If I try to delete the property using `delete`, I get `The operand of a 'delete' operator must be optional.`.

### Possible Solution
The type of `hitArea` could be set to `PIXI.IHitArea | null`, which would reflect its actual starting state. Alternatively, the property could be set to optional, using the `?` operator.

### Steps to Reproduce

```ts
let container = new PIXI.Container();
container.hitArea = new PIXI.Rectangle(0, 0, 10, 10);
container.hitArea = null;
```

### Environment

- **`pixi.js` version**: _6.2.1_
- **Browser & Version**: irrelevant
- **OS & Version**: irrelevant
- **Running Example**: -

**Comment by bigtimebuddy** [comment]
Good idea. PR welcome!

**Comment by Zyie** [comment]
Fixed in #8212
